### ISSUE TITLE
Allow PySCes_SBML to accept a file object

### ISSUE THREAD

**Issue by tbekolay** [robclewley/pydstool#7]
I'm working through the PyDSTool tutorials right now, and I'm very impressed! This is a great tool!

One thing that I tried to do and failed was to use a `StringIO` object instead of a file path to load an SBML file. I'm working in the IPython notebook, and I wanted everything to be in the same place, rather than having several files.

Here's my script:

``` python
import StringIO
from PyDSTool.Toolbox import PySCes_SBML

test_branch1 = StringIO.StringIO("""
# PySCeS test input file
# Branched pathway (2004)
FIX: x0 x5 x6
R0:
    x0 = s1
    Vf0*(x0 - s1/Keq0)/(x0 + KS0*(1 + s1/KP0))
R1:
    s1 = s2
    Vf1*(s1 - s2/Keq1)/(s1 + KS1*(1 + s2/KP1))
R2:
    s2 = s3
    Vf2*(s2 - s3/Keq2)/(s2 + KS2*(1 + s3/KP2))
R3:
    s2 = s4
    Vf3*(s2 - s4/Keq3)/(s2 + KS3*(1 + s4/KP3))
R4:
    s3 = x5
    Vf4*(s3 - x5/Keq4)/(s3 + KS4*(1 + x5/KP4))
R5:
    s4 = x6
    Vf5*(s4 - x6/Keq5)/(s4 + KS5*(1 + x6/KP5))

# InitExt
x0 = 10.0
x5 = 1.0
x6 = 1.0

# InitPar
Vf0 = 10.0
Vf1 = 10.0
Vf2 = 10.0
Vf3 = 10.0
Vf4 = 10.0
Vf5 = 10.0

Keq0 = 10.0
Keq1 = 10.0
Keq2 = 10.0
Keq3 = 10.0
Keq4 = 10.0
Keq5 = 10.0

KS0 = 5.0
KS1 = 5.0
KS2 = 5.0
KS3 = 5.0
KS4 = 5.0
KS5 = 5.0

KP0 = 1.0
KP1 = 1.0
KP2 = 1.0
KP3 = 1.0
KP4 = 1.0
KP5 = 1.0

# InitVar
s1 = 1.0
s2 = 1.0
s3 = 1.0
s4 = 1.0
""")
gen = PySCes_SBML.get_pysces_model(test_branch1, 'Vode')
```

This fails with the following traceback:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-5-c5a452db018d> in <module>()
     66 """)
     67 
---> 68 gen = PySCes_SBML.get_pysces_model(test_branch1, 'Vode')
     69 gen.set(tdata=[0, 10])
     70 gen.set(algparams={'init_step': 0.03})

/home/tbekolay/.virtualenvs/jneuro/lib/python2.7/site-packages/PyDSTool/Toolbox/PySCes_SBML.pyc in get_pysces_model(filename, target)
     77 
     78 def get_pysces_model(filename, target='Vode'):
---> 79     path, fname = os.path.split(filename)
     80     m = pysces.model(fname, dir=path)
     81 

/home/tbekolay/.virtualenvs/jneuro/lib/python2.7/posixpath.pyc in split(p)
     90     """Split a pathname.  Returns tuple "(head, tail)" where "tail" is
     91     everything after the final slash.  Either part may be empty."""
---> 92     i = p.rfind('/') + 1
     93     head, tail = p[:i], p[i:]
     94     if head and head != '/'*len(head):

AttributeError: StringIO instance has no attribute 'rfind'
```

Basically indicating that `get_pysces_model` expects a path instead of a file object. It seems like a common way to do this is to allow the function to accept either, and if it's passed in a path (or string-like) it loads it, otherwise it just uses the file object. I'm not sure if this is a `PySCes` limitation though.
